Tiltfile

```diff
@@ -140,6 +141,21 @@ else:
     target='compaction_service'
   )
 
+if config.tilt_subcommand == "ci":
```

Comment:
Also out of scope for this PR, but we should consider either a) having Depot do all our builds since its cache would be shared and thus lead to a faster build overall (I would imagine) or b) removing the conditional for whether we're running `tilt ci` or not. It feels weird that we have all these conditionals and that we are building differently for CI vs local.

Reply:
local builds against depot are slower because of network overhead